github/orgs_outside_collaborators_test.go

```diff
@@ -48,3 +48,81 @@ func TestOrganizationsService_ListOutsideCollaborators_invalidOrg(t *testing.T)
 	_, _, err := client.Organizations.ListOutsideCollaborators(context.Background(), "%", nil)
 	testURLParseError(t, err)
 }
+
+func TestOrganizationsService_RemoveOutsideCollaborator(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "DELETE")
+	}
+	mux.HandleFunc("/orgs/o/outside_collaborators/u", handler)
+
+	_, err := client.Organizations.RemoveOutsideCollaborator(context.Background(), "o", "u")
+	if err != nil {
+		t.Errorf("Organizations.RemoveOutsideCollaborator returned error: %v", err)
+	}
+}
+
+func TestOrganizationsService_RemoveOutsideCollaborator_NonMember(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "DELETE")
+		w.WriteHeader(http.StatusNotFound)
+	}
+	mux.HandleFunc("/orgs/o/outside_collaborators/u", handler)
+
+	_, err := client.Organizations.RemoveOutsideCollaborator(context.Background(), "o", "u")
+	if err, ok := err.(*ErrorResponse); !ok || err.Response.StatusCode != http.StatusNotFound {
+		t.Errorf("Organizations.RemoveOutsideCollaborator_NonMember returned error: %v", err)
```

Comment:
The `t.Errorf` error text here looks copied/pasted from above. But it doesn't accurately state what happened. It should be something like:

```Go
t.Errorf("Organizations.RemoveOutsideCollaborator did not return an error, or the status code wasn't 404 Not Found")
```